nft/nft.go

```diff
@@ -115,16 +124,32 @@ func (n NFT) IsValid([]byte) error {
 		return isvalid.InvalidError.Errorf("copyrighters over allowed; %d > %d", l, MaxCopyrighters)
 	}
 
+	foundSigner := map[base.Address]bool{}
```

Comment:
It is more recommended
`foundSigner := map[base.Address]struct{}{}`
because `struct{}` consumes no storage.

Reply:
@soonkuk I've addressed it!